### PR TITLE
ensure chat headers are visible

### PR DESCRIPTION
**Pull request by rcullito**
Fix for #4249 around visibility of chat headers.

### CODE REVIEW

File: src/status_im/ui/screens/desktop/main/chat/views.cljs [hunk 1]
```diff
@@ -23,12 +23,13 @@
...
-        (when public?
-          [icons/icon :icons/public-chat])
         (when (and group-chat (not public?))
           [icons/icon :icons/group-chat])
-        [react/text {:style {:font-size 16 :color :black :font-weight "600"}}
+        [react/text {:style {:font-size 16 :color :black :font-weight "300"}}
          chat-name]]
+       [react/view {:style {:flex-direction :row}}
+        (when public?
+          [react/text "Public Chat"])]
```
Comment (rcullito) [inline src/status_im/ui/screens/desktop/main/chat/views.cljs]: per: https://app.zeplin.io/project/5a324ae5d60d197e7e3369cc/screen/5b16ef8daeac41933f425aa9

**Review by vkjr — APPROVED** [review]

**Review by maxhora — APPROVED** [review]

**Comment by asemiankevich** [comment]
@rcullito please refer to any certain zeplin mock if any is implemented in PR

**Comment by rcullito** [comment]
@asemiankevich here is the most relevant link: https://app.zeplin.io/project/5a324ae5d60d197e7e3369cc/screen/5b16ef8daeac41933f425aa9

This PR does not completely match that design, but in particular I thought it would be convenient to add the text "public chat" while in the midst of fixing the "white panel issue" which this PR is meant to address. Let me know if any part of it is unclear! Thanks

<blockquote><div> Zeplin</div><div><strong><a href="https://zeplin.io/">Zeplin - Login</a></strong></div><div>Collaboration app for UI designers and frontend developers</div></blockquote>

**Comment by churik** [comment]
## 1. Chat headers are cut when you are switching between them (chat with longer name should be started first)

Steps:

1) log in
2) click on +
3) add smbd to contacts 
4) click on +
5) join any public channel, i.e. #status 
6) switch to  #status
7) switch to 1-1 chat. Pay attention to chat header. 

### Actual result:
can see only first 5 chars of contact name in Chat header

### Expected result: 
Contact name is fully displayed

**Note:**  is reporducible when on 3 step you join to any public chat with name longer than on 5 step
**Screen:** 
header
**Video:** http://take.ms/IZVNZ
**OS:**  Mac OS High Sierra

**Comment by churik** [comment]
## 2. Cannot see differences in chat headers (new design)

Also according to new design - not sure what was implemented.
Regarding public chat header, I can see that icon was removed, all other looks completely different from design mock.

long

**Comment by rcullito** [comment]
thanks for testing @churik. I will look into why the chat headers are cut as it relates to visibility issues brought up by #4249, which is what this PR aims to address.  

My aim was not to address all design feedback in this PR, but to open that up in a separate PR to come after these initial usability issues are fixed. I'll change the title of this PR to address that. Sorry for the confusion!

**Comment by rcullito** [comment]
@asemiankevich this latest commit should ensure that titles are always visible regardless of whether one is on initial page load or coming from a public chat to an individual one. 

There are still some issues of course plaguing the entire app around alignment, but hopefully this solves the visibility portion for now :pray: 

Thanks for looking into it!

**Comment by churik** [comment]
@rcullito 
1 issue (Chat headers are cut when you are switching between them (chat with longer name should be started first) - is still reproducible
Video: http://take.ms/gTvIa, [build](https://jenkins.status.im/job/status-react/job/desktop/job/pull-requests/view/change-requests/job/PR-4693/7/)

Also please request review from @cammellos or other mobile dev, it is required for now.
Thanks!

<blockquote><div>Monosnap screenshot tool</div><div><strong><a href="https://monosnap.com/file/17nzVJuwcpoXvNsB2RXfOF0p6sG6P0">File "screencast 2018-06-18 10-42-48.mp4"</a></strong></div><div>Monosnap — the best tool to take & share your screenshots.</div></blockquote>

**Comment by churik** [comment]
@rcullito 
please, take in mind that chat headers should be implemented according to design 
-  **1-1 chat:** zpl.io/V4J4NX4, zpl.io/am6gW4v
-  **Public chat:** zpl.io/2vlrMQe

**Comment by rcullito** [comment]
@churik thanks for the reminder on this! I'm aware that this is open/unresolved still. Let me know if I should indicate that in any way beyond being in the column: `contributor`. There are some other PRs that have landed in this general area so will take another look at this today updated agains the latest desktop branch 👍

**Comment by rcullito** [comment]
in recent talks it sounds like the fix for this is on the `react-native` side rather than achievable via styling tweaks. closing this PR for now.

**Pull request closed without merging**